smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/operation/input-output-traits/invalid-multiple-operations-same-shape.errors

```diff
@@ -1,4 +1,4 @@
-[WARNING] smithy.example#GetFoo2: The input of this operation should target a shape that starts with the operation's name, 'GetFoo2', but the targeted shape is `smithy.example#GetFooInput` | OperationInputOutputName
-[WARNING] smithy.example#GetFoo2: The output of this operation should target a shape that starts with the operation's name, 'GetFoo2', but the targeted shape is `smithy.example#GetFooOutput` | OperationInputOutputName
+[WARNING] smithy.example#GetFoo2: The input of this operation should target a shape that starts with the operation's name, 'GetFoo2', but the targeted shape is `smithy.example#GetFooInput` | OperationInputOutputName.input.smithy.example#GetFooInput
```

Comment:
I don't think the targeted input shape ID is useful in the event ID for this validator, so I'd say remove it.

Reply:
This was a case where I thought it necessary so that we don't unintendedly hide violations from a service team when something changes and the violation becomes slightly different. An example in this case would be where the operation changes to target another different shape that also does not start with the operation name. Ideally we _would_ show the violation in that case, even if the previous violation was approved or grandfathered in.

Reply:
That would get picked up by other validators, so I think it's a separate responsibility. This event is also demanding that the input/output has a name that matches the operation, so it seems very rare that you'd see this.

Reply:
Ok, will remove